### PR TITLE
chore: Update to V24.2 pre and add feature flag

### PR DESCRIPTION
**Pull request by mshabarov**
Explicit Form Filler dependency can be removed when we add the Form Filler to the platform.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +45,7 @@ You can play with the input text (remove parts, make a syntax mistakes) and see
 
 Integration tests are implemented using [Vaadin TestBench](https://vaadin.com/testbench). The tests take a few minutes to run and are therefore included in a separate Maven profile. We recommend running tests with a production build to minimize the chance of development time toolchains affecting test stability. To run the tests using Google Chrome, execute
 
-`mvn verify -Pit,production`
+`mvnw verify -Pit,production`
```
Comment (czp13) [inline README.md:48]: here as well `./mvnw...`

**Review by czp13 — COMMENTED** [review]
Small/Nitty question I guess to make it work with most of the operational systems we need to add more details in README.md.

```
To run the mvnw command, you may need to specify it differently depending on your operating system:

Windows: Use mvnw directly in your command prompt.
macOS/Unix: You may need to prepend ./ to the command, like this: ./mvnw.
```

Tested only Mac and it is `./mvnw` there.

File: README.md [hunk 2]
```diff
@@ -18,7 +23,7 @@ export OPENAI_TOKEN="THE KEY"
 ```
 - Windows: Use "System -> Advanced Settings -> Set Environment Variables" to set OPENAI_TOKEN
 
-The project is a standard Maven project. To run it from the command line, type `mvn` and open http://localhost:8080 in your browser.
+The project is a standard Maven project. To run it from the command line, type `mvnw` and open http://localhost:8080 in your browser.
```
[the remark this answers is not in the document]
Comment (mshabarov) [reply README.md:26]: Added a note about OS

**Comment by czp13** [comment]
Thank you very much for providing additional information about the environmental specifics! I see no other reason to not merge this PR. The added comments are just perfect in my eyes. 🙇

**Review by czp13 — APPROVED** [review]

**Pull request merged**